content/en/docs/reference/labels-annotations-taints/audit-annotations.md

```diff
@@ -72,6 +72,19 @@ violated from the PodSecurity enforcement.
 See [Pod Security Standards](/docs/concepts/security/pod-security-standards/)
 for more information.
 
+## apiserver.latency.k8s.io/validating-webhook
+
+Example: `apiserver.latency.k8s.io/validating-webhook: "203.698864ms"`
+
+This annotation shows the measure of the latency incurred in validating webhook(s).
```

Comment:
```suggestion
This annotation shows the measure of the latency incurred in validating webhook(s). Validating webhooks is done parallel, hence the latency is aggregated using max duration taken. 
```
I want to add this as the last line but don't know if this info is important to add here 
Please confirm

Reply:
Good to have this clarification.

Reply:
I'll add this in the next commit then